src/components/Logo.tsx

```diff
@@ -0,0 +1,34 @@
+import { Image, StyleSheet, useWindowDimensions, View } from 'react-native'
+
+import logo from '../../assets/images/logo.png'
+import Typography from './Typography'
+
+const styles = StyleSheet.create( {
+  headerTitle: {
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  logoIcon: {
+    width: 28,
+    height: 28,
+    marginRight: 6.5,
+  },
+  logoText: {
+    fontSize: 20,
+    lineHeight: 24,
+    fontWeight: '300',
+  },
+} )
+
+const Logo = () => {
+  const { width } = useWindowDimensions()
+
+  return (
+    <View style={styles.headerTitle}>
+      <Image style={styles.logoIcon} source={logo} />
+      {width > 350 && <Typography style={styles.logoText}>Shabad OS</Typography>}
```

Comment:
[nit] with these sorts of sizes, it might be better to collate them somewhere so that they're not magic numbers - that'll help future readers understand intent better